pkg/ibmvpc/vpc.go

```diff
@@ -577,20 +577,22 @@ type SecurityGroup struct {
 
 func (sg *SecurityGroup) AllowedConnectivity(src, dst vpcmodel.Node, isIngress bool) *connection.Set {
 	memberStrAddress, targetIPBlock := sg.getMemberTargetStrAddress(src, dst, isIngress)
+	_, localIPBlock := sg.getMemberTargetStrAddress(src, dst, !isIngress)
```

Comment:
better change the function `getMemberTargetStrAddress` to return `member.IPBlock()` as well, instead of calling it twice here